### ISSUE TITLE
Unable to delete public folder

### ISSUE THREAD

**Issue by mrvanes** [keybase/client#2035]
I mistyped my public foldername using my file explorer, which resulted in keybase creating the new public folder for me, but there doesn't seem to be a way (yet) to remove it (permission denied).

**Comment by cjb** [comment]
That's right, we don't have support for removing these folder "favorites" yet, but we will.  Sorry!

**Comment by hlapp** [comment]
@cjb is that true for mistakenly created folders under `/keybase/private/` as well, or is that unexpected and I should file as an issue? (BTW I'm not getting permission denied, but the folder doesn't disappear using `rmdir` or other means.)

**Comment by strib** [comment]
Oh, this issue is old.  `rmdir` is supposed to work on both public and private directories in the latest releases.  However, it will reappear in `ls` if you ever do `ls <folder_name>` or `cd <folder_name>`.  If that's not the behavior you're seeing, please do a `keybase log send` if you can, and give us more context about what you tried exactly.  Thanks!

**Comment by hlapp** [comment]
@strib OK, posted as #3253.

**Comment by obijan42** [comment]
TLDR: Every time there is an "ls" access on the "public" directory, it downloads upto 10 gig and there is nothing I can do about it.

Did I get that right?

**Comment by strib** [comment]
@obijan42: nope, that's not right.  When you `ls /keybase/public`, all it does is download your list of favorites from the Keybase server and display the names of those folders.  It doesn't download any data until you navigate into an individual folder and start looking around.

**Comment by obijan42** [comment]
`
ObiAir:public jhertsens$ rmdir chris
ObiAir:public jhertsens$ du -h .
1.3M	./chris/security_stuff
1.8M	./chris/Photos/this_is_a_dir/foo/bar/another_directory
1.8M	./chris/Photos/this_is_a_dir/foo/bar
1.8M	./chris/Photos/this_is_a_dir/foo
1.8M	./chris/Photos/this_is_a_dir
 13M	./chris/Photos
4.1G	./chris/verified_software/ubuntu-16.04
4.6G	./chris/verified_software
4.7G	./chris
4.0K	./obijan
4.7G	.
`
So you are saying this is a "heisenbug", that only exists when you look for it?

**Comment by strib** [comment]
After the `rmdir`, does an `ls` still show `chris`?  If so, you might be running an old version of Keybase -- if you want me to check it out, please do a `keybase log send`.

**Comment by obijan42** [comment]
Apparently it took 3 rmdir's in a row to remove it.
Success!
